src/tri_soft.c

```diff
@@ -712,9 +713,16 @@ void _al_triangle_2d(ALLEGRO_BITMAP* texture, ALLEGRO_VERTEX* v1, ALLEGRO_VERTEX
    }
 
    if (texture) {
+      ALLEGRO_BITMAP_WRAP wrap_u, wrap_v;
+      _al_get_bitmap_wrap(texture, &wrap_u, &wrap_v);
+
+      // XXX: Why are we using repeat for regular bitmaps by default?
```

Comment:
Indeed, why?

Reply:
I added more context. It's a bug I don't care to fix.

Reply:
Thanks for expanding that comment.